CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1782,12 +1783,24 @@ if userGuiIcon3D and vr3dGuis then
 	end
 end
 
+local function lookMenuEnabledChanged()
+	if InputService.VREnabled then
+		if lookMenuEnabled then
+			TopbarPanel3D:SetVisible(true)
+			--The look menu is very much tied to the backpack, so for now it will be responsible
+			--for showing/hiding it.
+			Panel3D.Get("Backpack"):SetVisible(StarterGui:GetCoreGuiEnabled(Enum.CoreGuiType.Backpack))
```

Comment:
Hm. It doesn't look like they'd fight, although it is kind of weird that panel:SetVisible happens here but the Backpack itself only toggles the guis.